internal/pkg/cli/svc_init.go

```diff
@@ -204,20 +208,34 @@ func newInitSvcOpts(vars initSvcVars) (*initSvcOpts, error) {
 	if err != nil {
 		return nil, fmt.Errorf("init a new local file selector: %w", err)
 	}
-
 	opts := &initSvcOpts{
-		initSvcVars:  vars,
-		store:        store,
-		fs:           fs,
-		init:         initSvc,
-		prompt:       prompter,
-		sel:          dfSel,
-		topicSel:     snsSel,
-		sourceSel:    sourceSel,
-		mftReader:    ws,
-		dockerEngine: dockerengine.New(exec.NewCmd()),
-		wsAppName:    tryReadingAppName(),
-		wsRoot:       ws.ProjectRoot(),
+		initSvcVars: vars,
+		store:       store,
+		fs:          fs,
+		init:        initSvc,
+		prompt:      prompter,
+		sel:         dfSel,
+		topicSel:    snsSel,
+		sourceSel:   sourceSel,
+		mftReader:   ws,
+		newAppVersionGetter: func(appName string) (versionGetter, error) {
+			return describe.NewAppDescriber(appName)
+		},
+		initEnvDescriber: func(appName string, envName string) (envDescriber, error) {
+			envDescriber, err := describe.NewEnvDescriber(describe.NewEnvDescriberConfig{
+				App:         appName,
+				Env:         envName,
+				ConfigStore: store,
+			})
+			if err != nil {
+				return nil, fmt.Errorf("initiate env describer: %w", err)
+			}
```

Comment:
is this context actually useful? or could we just do
```
return describe.NewEnvDescriber()
```

Reply:
oh mb forgot to push the commit